.pre-commit-config.yaml

```
@@ -24,25 +24,25 @@ repos:
       - id: mixed-line-ending
 
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: "v0.2.0"
+    rev: "v0.3.4"
     hooks:
       - id: ruff
         args: [--fix]
       - id: ruff-format
 
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.0.3
+    rev: v4.0.0-alpha.8
```

```suggestion
    rev: v3.0.3
```

This is a somewhat annoying quirk of the prettier mirror, that the latest release includes alpha preleases (or perhaps it is annoying quirk or prettier itself that these are not marked as proper pre-releases)